Makefile

```diff
@@ -64,13 +64,15 @@ INCLUDEOPTS=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
 	    | grep '^ /' | grep -v cc1plus | awk '{print "-I" $$1 " "}')
 PYINCLUDE=$(shell python -c \
 	  "import sysconfig;print(sysconfig.get_path('include'))")
+SYSPYINCLUDE=$(shell python -c \
+      "from __future__ import print_function; from distutils.sysconfig import get_python_inc; print(get_python_inc())")
```

Comment:
Do you have a link that explains when `SYSPYINCLUDE` and `PYINCLUDE` would be different? On my system they are the same and it seems odd that there should be two places to include things from. "One obvious way" and all that :-/